specification/edgemarketplace/resource-manager/readme.python.md

```diff
@@ -0,0 +1,35 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.edgemarketplace
+  package-name: azure-mgmt-edgemarketplace
+  package-version: 2023-04-01-preview
+  clear-output-folder: true
+```
+
+### Python multi-api
+
+Generate all API versions currently shipped for this package
+
+```yaml $(python) && $(multiapi)
+batch:
```

Comment:
please follow https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/samplefiles/readme.python.md to update `readme.python.md`

Reply:
fixed